Makefile

```diff
@@ -122,8 +122,8 @@ ifeq ($(TAG),)
 endif
 
 # Discover if user has dep installed -- prefer that
-DEP    := $(shell which dep    || echo "${ISTIO_BIN}/dep" )
-GOLINT := $(shell which golint || echo "${ISTIO_BIN}/golint" )
+DEP    := $(shell which dep    2>/dev/null || echo "${ISTIO_BIN}/dep" )
+GOLINT := $(shell which golint 2>/dev/null || echo "${ISTIO_BIN}/golint" )
```

Comment:
in theory we should use command I think instead of which (but maybe not in a makefile)